Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM centos:7.2.1511
+
+RUN yum -y install  gcc \
```

Comment:
I've never used Docker before (it looks pretty nice actually, I should give it a try). Unfortunately I have 32-bit  Ubuntu here, so I can't test if it works. 

Anyway, are you sure that configuration file you provided will run on systems other than Cent OS? It looks like it contains some distro-specific commands like `yum` here.